src/Servers/Kestrel/Core/src/Internal/HttpOutputProducerHelper.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Buffers
+{
+    internal class HttpOutputProducerHelper
+    {
+        public static IMemoryOwner<byte> ReserveFakeMemory(MemoryPool<byte> memoryPool, int sizeHint)
```

Comment:
Any reason why we can't rename `sizeHint` everywhere since it's not really a hint anymore, but a min?

Reply:
This one could be. The other ones would introduce a breaking change, or it would be done just on "some" methods that are private/internal.

Reply:
I like changing the name in non-public APIs.

Reply:
Changed it where internal and private were used